plugins/UFM_NDT_Plugin/ufm_sim_web_service/topo_diff/topo_diff.py

```diff
@@ -136,6 +146,104 @@ def parse_ndt_file(ndt_links, ndt_file, patterns, ndt_links_reversed=None):
+    # File created at   : 2022-08-25 17:46:20 UTC +0000
+    
+    
+    # Switch label port numbering explanation:
+    #   Quantum2 switch split mode: ASIC/Cage/Port/Split, e.g 1/1/1/1
+    #   Quantum2 switch no split mode: ASIC/Cage/Port
+    #   Quantum switch split mode: Port/Split
+    #   Quantum switch no split mode: Port
+    
+    
+    "MF0;DSM09-0101-0603-40IB1:MQM8700/U1", Mellanox, 0x1070fd03001adeb4, LID 24328
+      #          : IB# : Sta  : PhysSta    : MTU : LWA     : LSA     : FEC mode            : Retran : Neighbor Guid      : N#         : NLID : Neighbor Description
+      1          : 1   : ACT  : LINK UP    : 5   : 4x      : 50      : MLNX_RS_271_257_PLR : NO-RTR : 0xc42a10300dbfb92  : 40         : 25697 : "MF0;dsm09-0101-0602-01ib0:MQM8700/U1"
+      2          : 2   : ACT  : LINK UP    : 5   : 4x      : 50      : MLNX_RS_271_257_PLR : NO-RTR : 0xc42a10300e6c032  : 40         : 22524 : "MF0;dsm09-0101-0602-02ib0:MQM8700/U1"
+      3          : 3   : ACT  : LINK UP    : 5   : 4x      : 50      : MLNX_RS_271_257_PLR : NO-RTR : 0xc42a10300e6c57a  : 40         : 24891 : "MF0;dsm09-0101-0602-03ib0:MQM8700/U1"
+      4          : 4   : ACT  : LINK UP    : 5   : 4x      : 50      : MLNX_RS_271_257_PLR : NO-RTR : 0xc42a10300e6c3fa  : 40         : 33719 : "MF0;dsm09-0101-0602-04ib0:MQM8700/U1"
+      5          : 5   : ACT  : LINK UP    : 5   : 4x      : 50      : MLNX_RS_271_257_PLR : NO-RTR : 0xc42a10300e6c51a  : 40         : 21711 : "MF0;dsm09-0101-0602-05ib0:MQM8700/U1"
+      6          : 6   : ACT  : LINK UP    : 5   : 4x      : 50      : MLNX_RS_271_257_PLR : NO-RTR : 0xc42a10300e6c112  : 40         : 21400 : "MF0;dsm09-0101-0604-01ib0:MQM8700/U1"
+      7          : 7   : ACT  : LINK UP    : 5   : 4x      : 50      : MLNX_RS_271_257_PLR : NO-RTR : 0xc42a10300fca4a6  : 40         : 22400 : "MF0;dsm09-0101-0604-02ib0:MQM8700/U1"
+      8          : 8   : ACT  : LINK UP    : 5   : 4x      : 50      : MLNX_RS_271_257_PLR : NO-RTR : 0xc42a10300fcad46  : 40         : 23194 : "MF0;dsm09-0101-0604-03ib0:MQM8700/U1"
+    
+    """
+    links_list = []
+    links_info_dict = dict()
+    ibdiagnet_links = set()
+    ibdiagnet_links_reverse = set()
+    links_list_reversed = []
+    with open(net_dump_file_path, 'r') as f:
+        lines = f.readlines()
+        for line in lines:
+            if not line.strip():
+                continue
+            elif re.match(r'^#', line.strip()):
+                continue
+            elif re.match(r'^"', line):
+                # beginning of the switch
+                switch_info = line.split(",")
+                continue
+            else:
+                # lines of switch
+                link_info_list = line.split(":")
+                if link_info_list and link_info_list[2].strip().lower() == "down":
+                    continue
+                link_info_dict = {}
+                link_info_dict["node_name"] = switch_info[0].strip('"')
```

Comment:
better to use constants because you are using them in different places

Reply:
not in this case - there are a lot test fields there